### PR TITLE
Implemented a way to define Services + The Soap Service

### PR DESCRIPTION
**Pull request by BeeHiveJava**

### CODE REVIEW

File: web/src/main/java/nl/tehdreamteam/se42/web/soap/user/SoapUserController.java [hunk 1]
```diff
@@ -0,0 +1,16 @@
+package nl.tehdreamteam.se42.web.soap.user;
+
+import nl.tehdreamteam.se42.web.controller.UserController;
+
+import javax.jws.WebParam;
+import javax.jws.WebService;
+
+@WebService
+public class SoapUserController implements UserController {
```
Comment (TehMessiah) [inline web/src/main/java/nl/tehdreamteam/se42/web/soap/user/SoapUserController.java]: CRITICAL Document this public class. [rule](http://sonarqube.tehmessiah.nl/coding_rules#rule_key=squid%3AUndocumentedApi)

File: web/src/main/java/nl/tehdreamteam/se42/web/controller/UserController.java [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+    /**
+     * Logs a {@link User} in, given that the specified {@code username} and {@code password} are correct, and returns
+     * a {@code token} that temporarily identifies the logged-in {@code User}.
+     *
+     * @param username The {@code username} of the {@code User} to login.
+     * @param password The {@code password} of the {@code User} to login.
+     * @return A {@code token} that temporarily identifies the logged-in {@code User}.
+     */
+    String login(String username, String password); // TODO, should not be a String, probably a dedicated Token class
```
Comment (ghost) [inline web/src/main/java/nl/tehdreamteam/se42/web/controller/UserController.java:19]: INFO Complete the task associated to this TODO comment. [rule](http://sonarqube.tehmessiah.nl/coding_rules#rule_key=squid%3AS1135)

**Comment by TehMessiah** [comment]
SonarQube analysis reported 2 issues
* ![CRITICAL][CRITICAL] 1 critical
* ![INFO][INFO] 1 info

Watch the comments in this conversation to review them.

[CRITICAL]: https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL'
[INFO]: https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO'

**Comment by ghost** [comment]
SonarQube analysis reported 1 issue
* ![INFO][INFO] 1 info

Watch the comments in this conversation to review them.

[INFO]: https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO'

**Pull request merged**